packages/twenty-front/src/modules/ui/field/display/components/LinkDisplay.tsx

```diff
@@ -18,6 +18,12 @@ export const LinkDisplay = ({ value }: LinkDisplayProps) => {
     return <></>;
   }
 
+  const absoluteUrl = url
```

Comment:
we should leverage existing utils here :)

Reply:
Yes good point, might as well change it here too then, 
https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/modules/ui/field/display/components/URLDisplay.tsx#L32